script/glam/generate_sql

```diff
@@ -12,7 +12,7 @@ tables=$(bq ls --format=json $project:$dataset | \
 
```

Comment:
It would be worth using a filename like `generate_glean_sql` since we'll be adding a similar script for desktop.

Also dataset isn't specifiable via arg or env var.  Was that intentional?